Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -439,9 +439,11 @@ public void UseContentSecurityPolicy(IApplicationBuilder app)
             string connectSrc = cspSection.GetValue<string>("connect-src", string.Empty);
             string frameSrc = cspSection.GetValue<string>("frame-src", string.Empty);
             string scriptSrc = cspSection.GetValue<string>("script-src", string.Empty);
+            string styleSrc = cspSection.GetValue<string>("style-src", string.Empty);
+            string fontSrc = cspSection.GetValue<string>("font-src", string.Empty);
```

Comment:
this is kinda odd. If the weblcient does not need it, then admin probably doesn't need it either